build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/RestTestBasePlugin.java

```diff
@@ -75,6 +79,19 @@ public void apply(Project project) {
         project.getTasks()
             .withType(StandaloneRestIntegTestTask.class)
             .configureEach(t -> t.finalizedBy(project.getTasks().withType(FixtureStop.class)));
+
+        project.getTasks().withType(StandaloneRestIntegTestTask.class).configureEach(t ->
+        // if this a module or plugin, it may have an associated zip file with it's contents, add that to the test cluster
+        project.getPluginManager().withPlugin("elasticsearch.internal-es-plugin", plugin -> {
```

Comment:
We want to apply this logic to all inter tests tasks that are created in a module / plugin project